src/picongpu/CMakeLists.txt

```diff
@@ -253,6 +252,12 @@ if( (Boost_VERSION EQUAL 105500) AND
       "${CUDA_NVCC_FLAGS} \"-DBOOST_NOINLINE=__attribute__((noinline))\" ")
 endif()
 
+################################################################################
```

Comment:
ok for now, but as talked offline: we will simply add a include in the final version to reduce install dependencies.